clients/admin-ui/src/features/common/custom-fields/CreateCustomFields.tsx

```diff
@@ -49,30 +37,22 @@ const initialValues: CustomFieldDefinition = {
   resource_type: ResourceTypes.DATA_CATEGORY,
 };
 
-type FormValues = typeof initialValues;
```

Comment:
[Post-review]: Redundant alias for CustomFieldDefinition that was already being used.